types/black-engine/Engine.d.ts

```diff
@@ -47,9 +47,9 @@ export class Engine extends MessageDispatcher {
     private __bootStage;
     private __onVisibilityChangeFallback;
     __onVisibilityChange(): void;
-    hasSystem(systemTypeName: () => void): boolean;
+    hasSystem(systemTypeName: any): boolean;
```

Comment:
How come this one didn't become `(...params: any[]) => void` like the other callbacks?

Reply:
That's because it's not supposed to be a callback, it's a System type